guides/v2.1/javascript-dev-guide/widgets/widget_modal.md

```diff
@@ -71,7 +71,7 @@ Array of buttons for action pane.
 **Structure**:
 ```javascript
 buttons: [{
-    text: '',
+    text: $.mage.__(''),
```

Comment:
```suggestion
    text: '',
```

Reply:
Yes this one

Reply:
Hi @hardikm-cirkleinc 
Thanks for your contribution.
I think we don't need to do any change on line no 74.
Can you please revert change on line 74. we accept `$.mage.__('Ok')` this change.

Reply:
Hello @ronak2ram 
Thank you for your suggestion
I have changed line no 74 and I have also added this in line no 83.

Reply:
Can you please revert change on line 74.

Reply:
Hello @ronak2ram 

I have reverted

Reply:
Can you Please approve the changes

Reply:
I will approve soon. Can you please cover changes of PR #4702 in current PR. so we can decrease duplicate efforts.

Reply:
Done bro